spec.bs

```diff
@@ -2490,6 +2612,8 @@ The output of running a Protected Audience `scoreAd()` script, is represented us
 dictionary ScoreAdOutput {
   required double desirability;
   double bid;
+  DOMString bidCurrency;

```

Comment:
AuctionAdConfig's `sellerCurrency` is a `USVString` - do we want `bidCurrency` to be `DOMString`? (Maybe there is some meaningful difference and I'm missing it)

Reply:
Alright - works for me.

>  Is there a rule saying what AuctionAdConfig should be using, given that's clearly a usual web API?

Not that I'm aware of. But maybe @domfarolino has more knowledge than me on this.

Reply:
Guidance for string types can be found in https://webidl.spec.whatwg.org/#idl-USVString and especially https://w3ctag.github.io/design-principles/#idl-string-types. If you use USVString you do not have to implement a bad-surrogate-pairs check, as USVString does that _for you_. Maybe you're referring to the implementation though (which doesn't actually use WebIDL so we'd have to manually implement that check as WebIDL would automatically)?

So I think the question comes down to whether we are doing any text processing on the value and need to ever dig into the individual scalar values?

Reply:
It seems odd to be inconsistent here without good reason though. At the very least it would be good to document why these two currency types can be different, assuming them being different is the appropriate outcome.